container/run.sh

```diff
@@ -76,3 +76,24 @@ docker run -d \
 	$IMAGE_NAME
 
 echo "^ assigned container ID ^"
+
+container_id=$(docker ps -q -f name=pp)
+end_time=$((SECONDS+30))
+
+echo "Waiting for 'STANDBY AT PORT $HOSTPORT' in container logs..."
```

Comment:
@aacic should detect `'STANDBY AT PORT $EXPOSED_PORT'`. The HOSTPORT is for the URL that the user loads in his browser, the EXPOSED_PORT is the port that the express app listens to inside the container. 

For simplicity, we may force HOSTPORT to match the EXPOSED_PORT, but we'll need to document that `serverconfig.port` may be ignored inside the container if `serverconfig.url` is present and has a different port. Although, a user could also manually edit the serverconfig.port to match the port # in the serverconfig.url.

Reply:
@aacic We may also consider running an attached container instead of detached, so that they see the logs of when the server is ready/listening.